pages/1.11/release-notes/1.11.9/index.md

```diff
@@ -0,0 +1,122 @@
+- DCOS-46574 - Few tests do not have a `maintainer` metadata. Example:`test_ldap_sync.py` does not show a `maintainer` tag. This release enforces up-to-date maintainer tag on all tests. 
+- DCOS-46814 - When an agent host reboots, all of its containers fail but the agent will try to recover from its checkpoint state after reboot. The agent will soon discover that all the `cgroup` hierarchies have failed and assume that the containers are destroyed. However, when trying to terminate the executor, the agent will first try to wait for the `exit` status of its container by `waitpid` on the checkpointed child process `pid`. If an agent host reboots and a new process with the same `pid` gets spawned then the parent waits for the wrong child process. This process will block the executor termination and future task status updates. Bump Mesos to nightly 1.5.x 5f6225b.
+- DCOS_OSS-4040 - The `dcos-diagnostic` bundle files are stored in `/var/lib/dcos/dcos-diagnostics/diag-bundles` and `/var/lib/dcos` is not a separate partition. It belongs to the root directory and there are many other critical data in the same directory on the masters, which could seriously damage a cluster if diagnostics bundle takes most of the space. The workaround is to move the destination directory (read volume) from root partition to a different partition than all these other directories which allows `dcos-diagnostics` bundles location to be configured. 
+- DCOS_OSS-4531 - This issue is specific to the integration test suite. In one test, the fact is that all files in the diagnostics bundle are gzipped. But, Mesos only zips the files if they are over 1024 bytes big. Sometimes, the files would be empty or really small, thus resulting in a failure. In order to achieve correct test results, the tests are hardened to check for gzipped or non-gzipped files.
+
+## Metrics
+- DCOS_OSS-3863 - This release fixes a bug in `dcos-metrics` that caused Prometheus exporter to omit some metrics data on the agent nodes. 
+
+## Networking 
+- COPS-3743 - Erlang cookie serves as a membership token in a distributed system. Having it configurable would allow different DC/OS clusters to co-exist without interfering each other. The requirement is to have a unique and persistent Erlang cookie during the lifecycle for a given DC/OS cluster. This release uses `cluster_id` to satisfy the requirements.
+- COPS-4323, DCOS_OSS-4620 - Erlang has a concept of [cookie](http://erlang.org/doc/reference_manual/distributed.html#security) which allows/disallows a node to make a connection with the other nodes in the cluster. Only nodes with the same cookie strings are allowed to make connections. Currently, this cookie is a fixed string and not configurable in DC/OS. If nodes from different DC/OS clusters are reachable to each other then there is a possibility of "cross talk" between the two clusters. This fix is to make the cookie configurable.
```

Comment:
The first two items in this list are describing the same fix, I think they should be combined into one item.  The text in the second one ("Erlang has the concept of a cookie...") is very good,  I would suggest this version of the text be used for the combined item.